h/app.py

```diff
@@ -50,7 +51,14 @@ def includeme(config):
     config.add_request_method(in_debug_mode, "debug", reify=True)
 
     config.include("pyramid_jinja2")
-    config.add_jinja2_extension("h.jinja_extensions.Filters")
+    config.commit()
+    jinja2_env = config.get_jinja2_environment()
```

Comment:
This looks to me more "configuration" and avoids a class but happy to just add the `disable` on the class in `h.jinja_extensions` instead.

Reply:
I reverted this, it made the test a bit awkward